trussworks/react-uswds
# fix: Language selector: add prop for display lang for bug fix

# Summary

- Added a new optional prop to handle cases where the current language is persisted and the component should know which language to display. Defaults to setting the displayed language as the first item in the `langs` array.
- Broken out the drop down part of the selector into a separate file.
- Dropdown prefers to set the label with the new prop's label.

## Related Issues or PRs

closes #2621 

## How To Test

Use the new prop and confirm it overrides the existing behavior where the label is pulled from the first item in the `langs` array.





### Screenshots (optional)
<img width="251" alt="Screenshot 2023-10-06 at 6 36 05 AM" src="https://github.com/trussworks/react-uswds/assets/57681819/66dedf32-9894-43fb-b3d2-d34ac49fef0a">

<img width="112" alt="Screenshot 2023-10-06 at 6 36 26 AM" src="https://github.com/trussworks/react-uswds/assets/57681819/65bb9463-f629-4424-b1dd-15659390b82f">
